bin/docker/build

```diff
@@ -5,8 +5,9 @@
 # http://www.boost.org/LICENSE_1_0.txt)
 
 # Look for a suitable image name
-img="${BEAVER_DOCKER_IMG-$(git config hub.upstream)}"
-img="${img:-${TRAVIS_REPO_SLUG:-beaver}}"
+img="${BEAVER_DOCKER_IMG:-$(git config hub.upstream)}"
+img="${img:-$TRAVIS_REPO_SLUG}"
+img="${img:-beaver}"
```

Comment:
Can you split this into a separate PR so it can hit v0.4.0?

Reply:
You haven't noticed https://github.com/sociomantic-tsunami/beaver/pull/45/files , have you? :grumpy: